ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/clientlibs/editor/js/panelcontainer.js

```diff
@@ -116,13 +117,13 @@
             makeWrapDataReadOnly(this.value.length > 0);
         });
 
-        minInputField.addEventListener("change", function () {
+        /*minInputField.addEventListener("change", function () {
```

Comment:
we should remove the code instead of commenting

Reply:
we will be reverting this soon